firebase-acc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This admin key should probably not be public, if you could remove it from firebase and regenerate a new one that would be ideal.

Reply:
+1 I got hacked once even when my repo was private.

Heroku allows you to store config variables, so one copy will live there and one will live locally. Feel free to put something on my calendar if you want to go over this in more detail.